lib/manageiq/floe/workflow/runner/kubernetes.rb

```diff
@@ -3,6 +3,36 @@ module Floe
     class Workflow
       class Runner
         class Kubernetes < ManageIQ::Floe::Workflow::Runner
+          def initialize(*)
+            require "awesome_spawn"
+            require "securerandom"
+
+            super
+          end
+
+          def run!(resource, env = {}, secrets = {})
+            raise ArgumentError, "Invalid resource" unless resource&.start_with?("docker://")
+
+            image = resource.gsub("docker://", "")
+
+            name = "#{image.split("/").last.split(":").first}-#{SecureRandom.uuid}"
+            params = ["run", :rm, :attach, [:image, image], [:restart, "Never"], name]
+            params += env.map { |k, v| [:env, "#{k}=#{v}"] } if env && !env.empty?
```

Comment:
empty check isn't necessary - map will just return an empty array, and += will be a no-op